department-of-veterans-affairs/caseflow
# Appellant Info Redesign

Resolves #4233 

<img width="1052" alt="screen shot 2018-05-08 at 10 45 10 am" src="https://user-images.githubusercontent.com/4306128/39764116-e9047d88-52ac-11e8-86a3-04cd32f7ef71.png">

### Testing Plan
1. Go to the hearing worksheet. 
1. Confirm the UI displays according to the AC
1. Confirm the periods and circumstances of service, rep name, and witness save correctly
1. Confirm other hearing worksheet pages still display correctly
